ui/shared/components/panel/search/VariantSearchFormPanels.jsx

```diff
@@ -195,7 +195,7 @@ export const FREQUENCY_PANEL = {
   name: 'freqs',
   headerProps: {
     title: 'Frequency',
-    inputSize: 10,
+    inputSize: 12,
```

Comment:
the reported issue of the filter increasing from 9 to 1 is just that the input is too narrow to show a two digit number on some screens, increasing the size of the inputs should fix the issue